### DISCUSSION TITLE
Nominating @c0d1ngm0nk3y for a Contributor role

### DISCUSSION THREAD

**Discussion by ryanmoran** [paketo-buildpacks/feedback#126] [Node.js Team]
Hi @paketo-buildpacks/nodejs-maintainers,

I'd like to nominate @c0d1ngm0nk3y to join the Node.js contributor subteam. @c0d1ngm0nk3y has made a handful of contributions to the Node.js language family, mostly fixing issues with npm package management. You can see their contributions [here](https://github.com/pulls?q=is%3Apr+involves%3Ac0d1ngm0nk3y+user%3Apaketo-buildpacks+).

Can I get a 👍 / 👎 ?

**Comment by TisVictress** [comment]
👍 SGTM!!

**Comment by ryanmoran** [comment]
Ok. We have a majority in favor. Looks like this is approved. 🎉
